functions/Copy-SqlLogin.ps1

```diff
@@ -152,6 +152,12 @@ Limitations: Does not support Application Roles yet
 					continue
 				}
 				
+				if($destserver.LoginMode -ne "Mixed")
+				{ 
+					Write-Output "$Destination does not have Mixed Mode enabled. Enable this after the migration."
```

Comment:
This is probably open for debate, but I think this should be Write-Warning, to be consistent with the rest of the output.